apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -17,45 +17,54 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.regex.Matcher;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 /**
  * Represents the <code>manifest.xml</code> files that detail which specific files are included in a
  * transfer from the CMS Chronic Conditions Warehouse to the Blue Button API backend.
  */
+@NoArgsConstructor(force = true)
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(
     name = "",
     propOrder = {"entries", "preValidationProperties"})
 @XmlRootElement(name = "dataSetManifest")
 public final class DataSetManifest implements Comparable<DataSetManifest> {
   /** A timestamp {@link String} that maps to an S3 bucket folder. */
+  @Getter
```

Comment:
We should retain the setters here even if the xml parser doesn't require it.  Some tests might want to set these properties.

Reply:
Disregard this.